src/policy/file/v1.rs

```diff
@@ -195,38 +195,52 @@ impl KeyManagerSpec {
 
 impl Default for KeyManagerSpec {
     fn default() -> Self {
-        const ONE_DAY: u64 = 86400;
-        const FOUR_WEEKS: u64 = 2419200;
         Self {
+            // Default to KSK plus ZSK. CSK key rolls are more complex.
+            // No official reference.
             use_csk: false,
+
             algorithm: Default::default(),
-            ksk_validity: None, // Is this correct?
-            zsk_validity: None,
-            csk_validity: None,
+
+            // Roll a KSK once a year. No official reference.
+            ksk_validity: Some(365 * 24 * 3600),
+
+            // Roll a ZSK once a month. No official reference.
+            zsk_validity: Some(30 * 24 * 3600),
+
+            // Roll a CSK once a year just like a KSK. Assume that the DS
+            // record may need to be updated to hand.
+            csk_validity: Some(365 * 24 * 3600),
+
             auto_ksk: Default::default(),
             auto_zsk: Default::default(),
             auto_csk: Default::default(),
             auto_algorithm: Default::default(),
 
-            // Do we have a reference for this following durations?
-            dnskey_inception_offset: ONE_DAY,
-            dnskey_signature_lifetime: FOUR_WEEKS,
-            dnskey_remain_time: FOUR_WEEKS / 2,
-            cds_inception_offset: ONE_DAY,
-            cds_signature_lifetime: FOUR_WEEKS,
-            cds_remain_time: FOUR_WEEKS / 2,
+            // The following have roughly the same defaults as used for
+            // signing the zone.
+            dnskey_inception_offset: 24 * 3600,
+            dnskey_signature_lifetime: 14 * 24 * 3600,
+            dnskey_remain_time: 14 * 24 * 3600 / 2,
```

Comment:
Is this trying to say half of dnskey_signature_lifetime? That might be better expressed through a couple of related constants that can't accidentally diverge from one another.